tests/src/python/test_qgssymbollayer_readsld.py

```diff
@@ -444,7 +444,12 @@ def testLineOpacity():
         self.assertEqual(format.size(), 18)
         self.assertEqual(format.sizeUnit(), QgsUnitTypes.RenderPixels)
 
+        # the layer contains lines
+        # from qgis.core import QgsWkbTypes
+        # self.assertEqual(layer.geometryType(), QgsWkbTypes.LineGeometry)
```

Comment:
Why is this commented out?

Reply:
Because I'd like to verify that the layer contains line and the placement is line, but the placement gets from the layer's label settings is not line.